applications/dairdrop.md

```diff
@@ -0,0 +1,232 @@
+
+This community has continually considered and sought solutions to minimize expenses for a decentralized project, which has been a persistent goal of the Polkadot community. In the past, when some large token holders in the Dota community wanted to airdrop to community users, they felt frustrated because there was no simple tool to help them realize this idea.
+
+These challenges have hindered the development of decentralized projects. Polkadot has great technology, and community developers should take steps to lower the usage barriers for users in various aspects, making it accessible to more ordinary people not just a tool for technical experts and the wealthy.
+
+This application is designed for community projects that frequently need to conduct airdrops. It allows anyone to airdrop to their community or claim their own airdrop without relying on centralized servers, reducing dependence on centralized teams. 
+
+It can also mitigate the issue of airdropping to inactive accounts, as we know that airdropping to inactive accounts wastes valuable on-chain resources and can lead to permanent losses in on-chain liquidity. This phenomenon was particularly evident in DED's past MEME activities. Most importantly, it can unify airdrop standards on-chain, allowing users to monitor community airdrop activities based on on-chain transaction event information, thus improving transparency and enabling more users to participate in a timely manner.
+
+### Project Details
+
+- language `rust`（tauri）、`javascript`、`python`
+  > It will use the Tauri framework to achieve compatibility across multiple platforms, allowing it to be used on Windows, Linux, and macOS
+### Features
+
+- **Desktop APP**
+  - Connect to RPC nodes and create or connect wallets
+  - Support registering airdrop information on-chain (MD5 of the airdrop file, root of the Merkle tree for the airdrop accounts and amounts, start block, duration, URL of the airdrop file)
+  - Airdrop list (displaying airdrop status) and query airdrop status by user address
+  - Claim airdrops for any user (bulk claiming supported)
+  - Store airdrop data using Merkle tree
+  - Compatible with multiple operating systems(Linux、 Mac、 windows)
+
+- **Web Service**
+  - A web service with all the above functionalities
+
+- **An airdrop indexer**
+
+- **On-chain**
+  - An airdrop pallet(substrate pallet)
+    - Set airdrop: Merkle tree root, MD5 hash of the airdrop file, start date, end date, total airdrop amount (must be staked)
+    - Cancel airdrop
```

Comment:
> How will this work? Is the airdrop owned so that an account can decide on cancelling the airdrop at an arbitrary point in time?

When users decide to create an airdrop, they will register the airdrop information on-chain, including the block height at which the airdrop will occur, the root hash of the Merkle tree, the end time of the airdrop, and more. They will also need to reserve their balances to ensure that everyone can claim the airdrop. Additionally, they can terminate the airdrop activity at any time, as it is ultimately their token. This ensures flexibility for the airdrop.

Reply:
Why does the airdrop creator need to reserve any balances?

Reply:
> Why does the airdrop creator need to reserve any balances?

The airdrop creator reserve the airdrop total amount in dairdrop pallet to ensure that users can receive funds after their airdrop proof is verified. This is because the funds come from the airdrop creator. This ensures that the airdrop creator does not must to distribute the airdrop to each user individually. Anyone can claim the airdrop on behalf of any user, as long as they can provide the recipient's Merkle proof.